python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1646,17 +1645,16 @@ keys:
               keys:
                 source_protocol:
                   type: str
-                  # TODO AVD5.0: Add valid values and ensure we cover all options.
```

Comment:
"...and ensure we cover all options."

Did you do that?

Reply:
yes all options are covered apart from isis levels which is pending for all type of redistribute routes
```
wan1-dc1(config-router-bgp-af)#redistribute isis ?
  include    Include following routes while redistributing
  level-1    Redistribute IS-IS level-1 routes <-----------
  level-1-2  Redistribute IS-IS level-1 and level-2 routes <-------------
  level-2    Redistribute IS-IS level-2 routes <------------
  rcf        Routing control functions configuration to filter and modify paths
  route-map  Name a route map
```